app/lib/check_elastic_search.rb

```diff
@@ -82,6 +82,7 @@ def get_fresh_value(data)
 
   def add_update_nested_obj(options)
     return if self.disable_es_callbacks || RequestStore.store[:disable_es_callbacks]
+    return if options[:op] == 'create' && self.respond_to?(:hit_nested_objects_limit?) && self.hit_nested_objects_limit?
```

Comment:
Question: Only happens when the operation is "create"?

Reply:
yes as create add a new object to nested_object